Makefile

```diff
@@ -131,11 +148,35 @@ lint-static-check:
 lint: lint-static-check
 	$(LINT) run --timeout 5m
 
+.PHONY: multimod-verify
+multimod-verify: | $(MULTIMOD)
+	@echo "Validating versions.yaml"
+	$(MULTIMOD) verify
+
+COREPATH ?= "../opentelemetry-collector-contrib"
+.PHONY: multimod-sync-core
+multimod-sync-core: | $(MULTIMOD)
+	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
+	$(MULTIMOD) verify && $(MULTIMOD) sync -a -o ${COREPATH}
+
+
+.PHONY: multimod-prerelease
+multimod-prerelease: | $(MULTIMOD)
```

Comment:
We don't currently need this as we have no self dependencies. `aws-otel-collector` doesn't depend on `pkg/lambdacomponents` and vice versa.

Reply:
Reasonable. Changed the pkg/lambdacomponents to exclude modules.

Reply:
Wait, no keep the `pkg/lambdacomponents` in the module set. We just don't need to do the prerelease which is for updating the current version dependencies of the modules in the module set to the proposed new version. Like if the `aws-otel-collector` depended on the `pkg/lambdacomponents`, it would update the version in the go.mod. But we don't have that kind of dependency. So we just don't need the prerelease step.

Reply:
Instead of deleting the pre-release, would it be better to create another module set that includes ```pkg/lambdacomponents```? In this way, there will be no dependencies between the two and it will also preserve the ```prerelease``` functionality?

Reply:
From my understanding of `prerelease`, the only functionality we would get from it would be the creation of a branch named `prerelease-{module_set_name}-{new_version}` without any changes. I don't think that would be different if we had two module sets. Have you tried running it?

Reply:
Tested it with changing the version to v0.15.0 and not only creating a new branch but also update all the ```go.mod``` in the module sets ```adot-base```. 
![image](https://user-images.githubusercontent.com/91758108/141836031-94c701f5-3029-430c-a46a-3daa7068e123.png)

Reply:
What did it change in the go.mod?

Reply:
Right, so `prerelease` doesn't do anything for our use case.

Reply:
Wait.I still not quite clear. Cause the **go.mod is up to date** so there are no changes. But if the ```go.mod``` for the base-dot is not up-to-date, then I meant pre-release is still useful right? Also ```pre-release``` is the only command that update all go.mod in the module-sets. So I don't see any reason why ```pre-release``` is not useful